src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -260,6 +287,17 @@ public boolean accept(byte[] buf, int off, int len)
      */
     public void update(List<Endpoint> conferenceEndpoints, long bweBps)
     {
+        if (bweBps > -1)
+        {
+            if (lastBwe != -1 &&
+                Math.abs(bweBps - lastBwe) < lastBwe * BWE_THRES / 100)
```

Comment:
even though it's short, and probably only used here, i like the idea of helper functions for stuff like this...it makes reading it a bit clearer and makes it obvious where the adjustment needs to be made if we want to play with it, like:

```
public boolean isLargerThanBweThreshold(long currentBweBpsDelta, lastBweBps)
{
  return bweBpsDelta > lastBweBps * BWE_THRESHOLD_BPS / 100);
}
```
```
long bweBpsDelta = Math.abs(bweBps - lastBwe);
if (isLargerThanBweThreshold(bweBpsDelta)
{
  ...
```